src/back/gql/models/users/types.js

```diff
@@ -23,4 +28,20 @@ export const UserType = new GraphQLObjectType({
   },
 });
 
+export const CreateUserInputType = new GraphQLInputObjectType({
+  name: 'CreateUserInputType',
+  description: 'Input needed to create an User record',
+  fields: {
```

Comment:
Shouldn't this also include a **password** field as well as a **password validation**? Or should that be handled on the front end. Also, would password encryption be handled by the service?

*Note:* I'm not expecting us to validate within the time frame, just in case. This is just a question. The inclusion of the **password** field does feel relevant though.